detection_rules/rule.py

```diff
@@ -175,6 +176,14 @@ class BaseThreatEntry:
     name: str
     reference: str
 
+    @pre_load()
+    def modify_url(self, data: Dict[str, Any], **kwargs):
+        """Modify the URL."""
```

Comment:
it would be nice to have a short note here in the doc why the closing `/` is needed

Reply:
Good idea! Will add :+1: